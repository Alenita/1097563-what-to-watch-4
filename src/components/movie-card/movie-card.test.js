import React from "react";
import MovieCard from "./movie-card.jsx";
import renderer from "react-test-renderer";

const film = {
  poster: `img/poster-1.jpg`,
  title: `The Lord of the Rings: The Return of the King`,
};

it(`<MovieCard/> renders correctly`, () => {
  const tree = renderer
        .create(<MovieCard
          film={film}
          onHover={() => {}}
          onCardClick={() => {}}
        />)
            .toJSON();

  expect(tree).toMatchSnapshot();
});
