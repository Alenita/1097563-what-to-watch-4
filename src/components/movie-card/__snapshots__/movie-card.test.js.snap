// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCard/> renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="The Lord of the Rings: The Return of the King"
      height="175"
      src="img/poster-1.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      The Lord of the Rings: The Return of the King
    </a>
  </h3>
</article>
`;
